versioned_docs/version-3.0.0-beta/billing/plans.md

```diff
@@ -39,14 +54,38 @@ Users can upgrade or downsize their team in Spheron's Pro membership model to fi
 
 #### PRO Package
 
-Currently, the <b>Pro</b> subscription plan costs $10 per month per member. Covering in features like
-
-- <b>Unlimited Members, Unlimited Concurrent Builds</b>
-- <b>1000 minutes Build Execution</b>
-- <b>50 Domain Limit</b>
-- <b>50 ENS Domain Limit</b>
-- <b>50 HNS Domain Limit</b>
-- <b>2 Storage Package - 300MB Arweave, 4GB Skynet, 2GB Pinata, 2GB Filecoin</b>
+Currently, the <b>Pro</b> subscription plan costs starts from $20 per month per member. Covering in features like
+
+<b>Application</b>
+1. Concurrent Build → 3
+2. Build Execution → 6000 mins 
+3. Bandwidth → 1 TB 
+4. Decentralized domains → 10 
+5. IPFS Storage → 10 GB 
+6. Arweave → 1 GB 
+<b>Team</b>
+1. Member → 1 
+2. Unlimited Preview Links → Enabled
+3. GitHub / Gitlab / Bitbucket Support → Enabled
+4. Private Repository Support → Enabled
+<b>Business</b>
+1. Domains → Unlimited
+2. Deployment per day → 1000
+3. Notification → 20k per month
+4. IPFS & Filecoin support → Enabled
+5. Arweave support → Enabled
+6. Shared Spheron IPFS Gateway → Enabled
+7. Analytics → $10
```

Comment:
remove this for now, we don't have this yet

Reply:
Updated